client-src/index.js

```diff
@@ -133,11 +133,17 @@ const onSocketMessage = {
 
     reloadApp(options, status);
   },
+  // TODO: remove in v5 in favor of 'static-changed'
   'content-changed': function contentChanged() {
     log.info('Content base changed. Reloading...');
 
     self.location.reload();
   },
+  'static-changed': function staticChanged() {
+    log.info('Contents from static directory were changed. Reloading...');
```

Comment:
Hm, maybe we can pass file which was changed to it will be easy to debug for developers and less questions why page was reloaded?

Reply:
Yes, we can, should we backport this change to `content-changed` as well?

Reply:
Yep, should be easy

Reply:
Done